ui/hooks/src/moderation-request.ts

```diff
@@ -255,7 +255,7 @@ export default {
             throw new Error('Unable to process your request right now. Please try again later');
           }
 
-          return modalName === 'report-modal' ? callback(true) : callback();
+          return modalName === 'report-modal' ? callback(true, dataToSign.reason) : callback();
```

Comment:
I would suggest to always pass those parameters to the callback and get rid of this logic here, and remove the unnecessary modalName param.